app/assets/stylesheets/app/forms.css.scss

```diff
@@ -0,0 +1,3 @@
+label.disabled {
+  color: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.